css/_components/modal/_modal.css

```diff
@@ -0,0 +1,139 @@
+.modal__container {
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.2);
+  display: none;
+  height: 100vh;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100vw;
+}
+
+.modal {
+  background-color: white;
+  border-radius: var(--br-sm);
+  box-shadow: var(--bs-md);
+  flex-direction: column;
+  padding: var(--pd-sm);
+  width: min(85%, 500px);
+  min-width: 350px;
+}
+
+.modal__footer {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 20px;
+}
+
+.modal__footer--btn {
+  margin-right: var(--space-sm);
+}
+
+.modal__footer--btn:last-child {
+  margin-right: 0;
+}
+
+/* W/ICONS */
+.modal__icons {
+  padding: 0;
+}
+
+.modal__icons .modal__title,
+.modal__icons .modal__body,
+.modal__icons .modal__footer {
+  padding: var(--pd-sm);
+}
+
+.modal__title--close:hover {
+  background-color: initial;
+}
+
+.modal__icons .modal__title {
+  align-items: center;
+  border-bottom: 1px solid var(--grey-1);
+  display: flex;
+  justify-content: space-between;
+}
+
+.modal__icon .modal__footer {
+  margin-top: 0px;
+}
+
+/* PRIMARY */
+.md-primary .modal__title {
+  background-color: var(--primary);
+  border-top-left-radius: var(--br-sm);
+  border-top-right-radius: var(--br-sm);
```

Comment:
These two styles are repeating in various classes, they can be added in a separate class

Reply:
Sure, will do the changes 👍  
Thanks for pointing it out